mentor_classifier/tests/test_train_and_predict.py

```diff
@@ -27,48 +32,51 @@ def shared_root(word2vec) -> str:
 
 @responses.activate
 @pytest.mark.parametrize(
-    "mentor_id,expected_training_accuracy,sample_questions,expected_sample_answers",
+    "training_configuration",
     [
-        (
-            "clint",
-            0.5,
-            [
-                "What's your name?",
-                "Who are you?",
-                "Give me your name.",
-                "Give me your age.",
-                "what do the penguins do?",
-            ],
-            [
-                "Clint Anderson",
-                "Clint Anderson",
-                "Clint Anderson",
-                "37 years old",
-                "Penguins do important things",
-            ],
+        _MentorTrainAndTestConfiguration(
+            mentor_id="clint", arch=ARCH_LR, expected_training_accuracy=0.5
         )
     ],
 )
 def test_train_and_predict(
-    mentor_id: str,
-    expected_training_accuracy: float,
-    sample_questions: List[str],
-    expected_sample_answers: List[str],
+    training_configuration: _MentorTrainAndTestConfiguration,
     tmpdir,
     shared_root: str,
 ):
-    mentor = load_mentor_csv(fixture_path("csv/{}.csv".format(mentor_id)))
+    mentor = load_mentor_csv(
+        fixture_path(
+            f"csv/{training_configuration.mentor_id}/{training_configuration.mentor_id}.csv"
```

Comment:
one thing I didn't notice in last comment:

These are paths, so use `os.path.join("csv", training_configuration.mentor_id, f"{training_configuration.mentor_id}.csv")`

...same for instance below. We should manually assemble paths (assume the / separator)